backend/test/controllers/OdinsonControllerSpec.scala

```diff
@@ -183,6 +183,17 @@ class OdinsonControllerSpec extends PlaySpec with GuiceOneAppPerTest with Inject
       println(Helpers.contentAsString(response))
     }
 
+    "respond with frequency table using the /term-hist endpoint" in {
+      val body = Json.obj("field" -> "lemma")
+
+      val response =
+        route(app, FakeRequest(GET, "/api/term-hist?field=lemma").withJsonBody(body)).get
```

Comment:
Can we add another test that includes some of the other optional params for this endpoint (ex. log scale)?